Binary Search Tree/Merge_Two_BST.cpp

```diff
@@ -0,0 +1,184 @@
+// { Driver Code Starts
```

Comment:
Try to include a comment containing the **Problem statement** with **Sample Test Cases** at the very top of the solution, that has been tested with the code for future `PRs`. 👨🏻‍💻
Refer : [CONTRIBUTING.md](https://github.com/akashchouhan16/Data_Structures_And_Algorithms/blob/master/CONTRIBUTING.md "View Guidelines")